supertokens_python/recipe/thirdpartyemailpassword/interfaces.py

```diff
@@ -29,6 +15,28 @@
 ThirdPartyAPIOptions = ThirdPartyInterfaces.APIOptions
 EmailPasswordAPIOptions = EPInterfaces.APIOptions
 
+# Exporting re-used classes
+CreateResetPasswordOkResult = EPInterfaces.CreateResetPasswordOkResult
+CreateResetPasswordWrongUserIdErrorResult = EPInterfaces.CreateResetPasswordWrongUserIdErrorResult
+EmailExistsGetOkResponse = EPInterfaces.EmailExistsGetOkResponse
```

Comment:
This should be prefixed with EmailPassword since they are outputs from functions that also have the prefix of email_password